com.unity.render-pipelines.universal/Documentation~/lit-shader.md

```diff
@@ -49,6 +49,7 @@ The __Surface Inputs__ describe the surface itself. For example, you can use the
 | __Metallic / Specular Map__ | Shows a map input for your chosen __Workflow Mode__ in the __Surface Options__.<br/>For the [__Metallic Map__](https://docs.unity3d.com/Manual/StandardShaderMaterialParameterMetallic.html) workflow, the map gets the color from the __Base Map__ assigned above. Use the slider to control how metallic the surface appears. 1 is fully metallic, like silver or copper, and 0 is fully dielectric, like plastic or wood. You can generally use values in between 0 and 1 for dirty or corroded metals.<br/>For the [__Specular Map__](https://docs.unity3d.com/Manual/StandardShaderMaterialParameterSpecular.html) setting, you can assign a texture to it by clicking the object picker next to it. This opens the Asset Browser, where you can select from the Textures in your Project. Alternatively, you can use the [color picker](https://docs.unity3d.com/Manual/EditingValueProperties.html).<br/>For both workflows, you can use the __Smoothness__ slider to control the spread of highlights on the surface. 0 gives a wide, rough highlight. 1 gives a small, sharp highlight like glass. Values in between produce semi-glossy looks. For example, 0.5 produces a plastic-like glossiness.<br/>Use the __Source__ drop-down menu to select where the shader samples a smoothness map from. Options are: __Metallic Alpha__ (alpha channel from the metallic map), and __Albedo Alpha__ (alpha channel from the base map). The default value is __Metallic Alpha__.<br/>If the selected source has the alpha channel, the shader samples the channel and multiplies each sample by the __Smoothness__ value. |
 | __Normal Map__              | Adds a normal map to the surface. With a [normal map](https://docs.unity3d.com/Manual/StandardShaderMaterialParameterNormalMap.html?), you can add surface details like bumps, scratches and grooves. To add the map, click the object picker next to it. The normal map picks up ambient lighting in the environment. <br/>The float value next to the setting is a multiplier for the effect of the  __Normal Map__. Low values decrease the effect of the normal map. High values create stronger effects. |
+| __Height Map__              | Adds a height map to the surface. The [height map](https://docs.unity3d.com/Manual/StandardShaderMaterialParameterHeightMap.html?) is used with parallax mapping technique to archieve surface-level occlusion effect by shifting the areas of the visible surface texture. To add the map, click the object picker next to it. <br/>The float value next to the setting is a multiplier for the effect of the  __Height Map__. Low values decrease the effect of the height map. High values create stronger effects. |
```

Comment:
Change this:
"The [height map](https://docs.unity3d.com/Manual/StandardShaderMaterialParameterHeightMap.html?) is used with parallax mapping technique to archieve surface-level occlusion effect by shifting the areas of the visible surface texture."
To this:
"URP implements the parallax mapping technique which uses the [height map](https://docs.unity3d.com/Manual/StandardShaderMaterialParameterHeightMap.html?) to achieve surface-level occlusion effect by shifting the areas of the visible surface texture."